components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/behavior/BrowserToolbarBehavior.kt

```diff
@@ -37,27 +39,20 @@ private const val SMALL_ELEVATION_CHANGE = 0.01f
  * - Snap the [BrowserToolbar] to be hidden or visible when the user stops scrolling.
  */
 @Suppress("TooManyFunctions")
-class BrowserToolbarBottomBehavior(
+class BrowserToolbarBehavior(
     val context: Context?,
-    attrs: AttributeSet?
+    attrs: AttributeSet?,
+    private val toolbarPosition: ToolbarPosition
```

Comment:
With that I assume you can no longer use this behavior from XML? But it's also not important to keep. :)

Reply:
Yes, currently we don't use it in xml so I skipped this part.
I could add an attribute for it later if needed?
Having in one place the code for both toolbars was the most important thing for me.

Reply:
👍